merge.go

```diff
@@ -275,6 +275,15 @@ func loadMergeFiles(dirPath string) error {
 	// should be moved to the original data directory, and the original data files should be deleted.
 	for fileId := uint32(1); fileId <= mergeFinSegmentId; fileId++ {
 		destFile := wal.SegmentFileName(dirPath, dataFileNameSuffix, fileId)
+		_, err = os.Stat(destFile)
```

Comment:
```
if _, err = os.Stat(destFile); os.IsNotExist(err) {
   contine
}
```
this will be more clear?

Or you can add a utility function named `FileExists`(or something like this) in util/file.go.

BTW, we just skip to remove the file, but I also want to make sure is there any other logic bugs in this case?
Thanks.

Reply:
ok, I'll add a unit test